@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -631,13 +632,33 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
             const computeTexCoords = this.getComputeTexCoordsFunc(technique);
             const vertexStride = computeTexCoords !== undefined ? 4 : 2;
 
+            let clipRing: THREE.Vector2[] | undefined;
+
             for (const polygon of geometry) {
                 const rings: Ring[] = [];
 
                 for (const outline of polygon.rings) {
                     const ringContour: number[] = [];
 
-                    for (const coord of outline) {
+                    let coords = outline;
+
+                    if (!isExtrudedPolygonTechnique(technique)) {
+                        if (!clipRing) {
+                            clipRing = [
+                                new THREE.Vector2(0, 0),
+                                new THREE.Vector2(extents, 0),
+                                new THREE.Vector2(extents, extents),
+                                new THREE.Vector2(0, extents)
+                            ];
+                        }
+                        coords = clipPolygon(coords, clipRing);
```

Comment:
Maybe I missed it but I cannot see any code handling the "pseudo" edges introduced by clipping. Will it work if we render the polygon with outline or will we see an outline on the tile border?
An image regression test for that case would also be highly appreciated.

Reply:
The way stroking is implemented is just not working reliably when tiles are clipped and we have sources where tiles are clipped. If we want to support stroking polygons we may need to rewrite that code, maybe using a similar approach as the one we use for outlines.